### ISSUE TITLE
accessing vcov in conleyreg

### ISSUE THREAD

**Issue by zhawboku** [ChristianDueben/conleyreg#3]
Hello Christian, 
I want to derive marginal effect estimates for some model variables I use to estimate a Conley HAC regression using your package conleyreg. But I can't figure out how to access  variance-covariance matrix after running regression. 

In your vignette, you specify the option vcov = TRUE. I use it but wonder whether it is saved then.
Could you clarify it to me?

Thanks, 
Raushan

**Comment by cdueben** [comment]
Hi Raushan,

The function returns the variance-covariance-matrix, when `vcov = TRUE`.

```
data <- rnd_locations(100, output_type = "data.frame")
data$y <- sample(c(0, 1), 100, replace = TRUE)
data$x1 <- stats::runif(100, -50, 50)
vcovmatrix <- conleyreg(y ~ x1, data, 1000, lat = "lat", lon = "lon", vcov = T)
```

`vcovmatrix` then holds that matrix.

Kind regards,
Christian

**Comment by zhawboku** [comment]
Many thanks, Christian.


On 8 Oct 2024, at 03:38, Christian Düben ***@***.***> wrote:

﻿

Closed #3<https://github.com/cdueben/conleyreg/issues/3> as completed.
